sklearn/linear_model/tests/test_logistic.py

```diff
@@ -492,6 +493,37 @@ def test_logistic_cv():
     assert_array_equal(scores.shape, (1, 3, 1))
 
 
+@pytest.mark.parametrize('scoring', ['accuracy', 'f1', 'neg_log_loss',
+                                     'precision', 'recall'])
+def test_logistic_cv_multinomial_score(scoring):
+    # test that LogisticRegressionCV uses the right score to compute its
+    # cross-validation scores when using a multinomial scoring
```

Comment:
This only fails in master for `neg_log_loss`, which makes sense in terms of the fact that OVR and Multinomial should find similar categorical predictions, but different probabilities. I think testing all the different scorers is a red herring.

Reply:
Why would they find similar categorical predictions ? Isn't it what this example is tackling ? http://scikit-learn.org/stable/auto_examples/linear_model/plot_logistic_multinomial.html (and there are clearly different decision boundaries)
But however indeed the test was only failing on master for `neg_log_loss`, so maybe I should test on an example that would fail for all multinomial scores ?

Reply:
Oh, now I understand, the decision boundaries in the example are different because of the fitting part which has a different objective between 'ovr' and 'multinomial'. But at prediction time, `decision_function` is the same for both so only `predict_proba` will change but not `predict` (as `predict_proba` keeps the ordering from `decision_function` results in the two cases), indeed...
So yeah I agree testing for all scores is a red herring. Should I remove them from the test and keep only `neg_log_loss` ? or test directly `predict_proba` ?